src/ferc_xbrl_extractor/cli.py

```diff
@@ -9,9 +9,15 @@
 
 from ferc_xbrl_extractor import helpers, xbrl
 from ferc_xbrl_extractor.helpers import get_logger
-from ferc_xbrl_extractor.instance import Instance
+from ferc_xbrl_extractor.instance import InstanceBuilder
 
-DEFAULT_TAXONOMY = "https://eCollection.ferc.gov/taxonomy/form1/2022-01-01/form/form1/form-1_2022-01-01.xsd"
+TAXONOMY_MAP = {
+    1: "https://eCollection.ferc.gov/taxonomy/form1/2022-01-01/form/form1/form-1_2022-01-01.xsd",
+    2: "https://eCollection.ferc.gov/taxonomy/form2/2022-01-01/form/form2/form-2_2022-01-01.xsd",
+    6: "https://eCollection.ferc.gov/taxonomy/form6/2022-01-01/form/form6/form-6_2022-01-01.xsd",
+    60: "https://eCollection.ferc.gov/taxonomy/form60/2022-01-01/form/form60/form-60_2022-01-01.xsd",
+    714: "https://eCollection.ferc.gov/taxonomy/form714/2022-01-01/form/form714/form-714_2022-01-01.xsd",
```

Comment:
Does FERC plan to release taxonomies that aren't just integers?

Reply:
Also, will these taxonomies change? Are we archiving them?

Reply:
I'm not really sure if they will ever have any non-integer forms to worry about, but as of now they all are, so I just did it this way for simplicity.

The taxonomies do change, but it would be a new version that would be have a different URL. Every XBRL filing contains a reference to the taxonomy that was used to construct it, so the URL's here _should_ point to the 2022 version of the taxonomy forever. That being said, we probably should archive the taxonomies just to be safe.